operator/src/main/java/org/keycloak/operator/controllers/KeycloakServiceMonitorDependentResource.java

```diff
@@ -46,6 +47,7 @@ protected ServiceMonitor desired(Keycloak primary, Context<Keycloak> context) {
               .withNewMetadata()
                 .withName(meta.getName())
                 .withNamespace(meta.getNamespace())
+                .withLabels(Utils.allInstanceLabels(primary))
```

Comment:
@ryanemerson this appears to be missing from the original servicemonitor pr. It showed up in some test failure here that creating the servicemonitor was failing because it already existed.

Let me know if you want this in a different pr.